tests/test_main.py

```diff
@@ -37,4 +37,4 @@ def test_main__argparse__invalid_arg():
 def test_main__logging__writes_to_stderr(patched_controller, capsys):
     main([])
     _, err = capsys.readouterr()
-    assert "INFO:	Starting Seedsigner with: {'loglevel': 'INFO'}" in err
+    assert "     INFO [main main (46)]: Starting Seedsigner with: {'loglevel': 'INFO'}" in err
```

Comment:
Minor nit: I'd just assert some key phrases from the expected output. It'll be more maintainable if the test isn't so tightly coupled to the exact log output.

```python
assert "Starting Seedsigner" in err and "INFO" in err
```

Reply:
done. in https://github.com/SeedSigner/seedsigner/pull/558/commits/d088fbdb42cba66cba848e70c5da959a171d1624